Zend/zend_language_scanner.l

```diff
@@ -631,6 +634,27 @@ zend_op_array *compile_filename(int type, zval *filename)
 		convert_to_string(&tmp);
 		filename = &tmp;
 	}
+	if (!CG(script_extensions)[0]) {
```

Comment:
AFAIK streams like phar, zlib, etc. aren't marked as is_url so allow_url_include doesn't work on all of them. I'll try to look for specific way around it later.